eng/pipelines/libraries/helix-queues-setup.yml

```diff
@@ -85,6 +85,10 @@ jobs:
         # Limiting interp runs as we don't need as much coverage.
         - Debian.9.Amd64.Open
 
+    # OSX arm64
+    - ${{ if eq(parameters.platform, 'OSX_arm64') }}:
```

Comment:
does this need to be scoped to `outerloop` or something so it doesnt trigger for PR jobs just yet?

Reply:
or if we want it on CI use `eq(parameters.jobParameters.isFullMatrix, true)`

Reply:
So this doesn't trigger jobs per se.  It should define the helix queues for a platform.

I am not sure this is the appropriate place to limit the usage of the platform queues.  When I had `OSX_arm64` in the `platformGroup` `all` there were flames everywhere because the helix-queues weren't defined.

I suspect we need to gate the jobs elsewhere.  Possibly in the platform-matrix.  Perhaps `OSX_arm64` should only be in the `all` group during `ci` runs.

Reply:
Yeah we usually limit the usage of helix queues for a platform here. However, I just realized even if we make this change at the moment, tests wont run in that queue as the jobs that submit the test jobs don't have `OSX_arm64` in their platform list. I think we should make this addition to the helix queue list once we actually add test runs for `OSX_arm64`. 

When we do that we should first know if we have machine capacity to enable it for PRs, or only for runtime tests and then for libraries, and so on. 

cc: @steveisok as he is experimenting with this as well.

Reply:
OK.  I'll enable the CI tests in this PR as well.

Initially I expect the runs to have a lot of intermittent failures, so I don't want to enable them for PRs even if we have capacity.

Are we OK with osx-arm64 CI failures for a while while we collect lists of failing tests.  Or should I set something like `shouldContinueOnError`?

Reply:
Yeah enabling CI tests should be fine. Do failures in a `CI` job cause any consequences like the build not getting published to nuget feeds etc?

Reply:
> Are we OK with osx-arm64 CI failures for a while while we collect lists of failing tests. Or should I set something like shouldContinueOnError?

You could instead add them to `runtime-staging` pipeline which it's intent is to add new platforms that are not stable. 

Another approach is disable all failures without merging. However in the `runtime` pipeline we can't merge with any failures as that pipeline we track it's pass rate. ContinueOnError might be another approach if adding to `runtime-staging` is too complicated.

Note that it seems like the Helix M1 queues don't have any machines yet. At least from what I can see on the kusto queries.

Reply:
/cc @steveisok